{"name": "test_001", "status": "failed", "statusDetails": {"message": "AssertionError: assert 0", "trace": "self = <scripts.test_case.TestCase object at 0x000001F372BCF730>\n\n    def test_001(self):\n>       assert 0\nE       assert 0\n\nscripts\\test_case.py:3: AssertionError"}, "start": 1748149116799, "stop": 1748149116799, "uuid": "3606b7ad-b916-4245-b5bd-66a17b0fb5a9", "historyId": "b52620e2fa88e5ec1efe61251c01560f", "testCaseId": "b52620e2fa88e5ec1efe61251c01560f", "fullName": "scripts.test_case.TestCase#test_001", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test_case"}, {"name": "subSuite", "value": "TestCase"}, {"name": "host", "value": "LAPTOP-47HNDVHS"}, {"name": "thread", "value": "30256-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test_case"}]}